src/Illuminate/Foundation/Console/Tinker/Presenters/EloquentCollectionPresenter.php

```diff
@@ -0,0 +1,31 @@
+<?php namespace Illuminate\Foundation\Console\Tinker\Presenters;
+
+use Psy\Presenter\ObjectPresenter;
+use Illuminate\Database\Eloquent\Collection;
+
+class EloquentCollectionPresenter extends ObjectPresenter {
+
+	/**
+	 * EloquentCollectionPresenter can present Collections.
+	 *
+	 * @param mixed $value
+	 * @return boolean
+	 */
+	public function canPresent($value)
+	{
+		return $value instanceof Collection;
+	}
+
+	/**
+	 * Get an array of Collection object properties.
+	 *
+	 * @param object           $value
+	 * @param \ReflectionClass $class
```

Comment:
There's actually inconsistency in the Laravel codebase on this, so do you mind pointing me to one that's done correctly or letting me know exactly what I'm doing wrong here? Which of these is correct, or, if neither, what would be?

```php
/**
 * @param  type  $name  Description
 * @param  longertype  $longerName  Longer description
 * @return type
 */
```
```php
/**
 * @param  type        $name        Description
 * @param  longertype  $longerName  Longer description
 * @return type
 */
```

Thanks!